pkg/skaffold/build/jib/build.go

```diff
@@ -34,15 +34,15 @@ func (b *Builder) Build(ctx context.Context, out io.Writer, artifact *latest.Art
 	switch t {
 	case JibMaven:
 		if b.pushImages {
-			return b.buildJibMavenToRegistry(ctx, out, artifact.Workspace, artifact.JibArtifact, tag)
+			return b.buildJibMavenToRegistry(ctx, out, artifact.Workspace, artifact.JibArtifact, artifact.Dependencies, tag)
```

Comment:
I do wonder if it wouldn't be simpler to just pass in `artifact` and letting the `build*` functions indirect through `.JibArtifact`? It would saves us from more churn if we add more fields to the artifact.

Not something necessary in this PR unless you feel like it.